### DISCUSSION TITLE
Need to have Gaussian software installed prior installing RMG

### DISCUSSION THREAD

**Discussion by tsn2027** [ReactionMechanismGenerator/RMG-Py#2629] [General]
Hello all,

I wonder whether I need to have Gaussian software installed prior installing RMG to calculate thermo properties of the species I am looking for?

**Comment by JacksonBurns** [comment]
@tsn2027 Gaussian is not required to run RMG. It and other simulation tools like MOPAC can be used to run some calculations on the fly, but are not a necessity.

**Comment by tsn2027** [comment]
Thank you! 

I know that I can submit SMILES string on RMG website to obtain thermodynamics properties right away; however, I tried to do that with Arkane (without providing any gaussian data) but it it impossible. Or I might miss something. Thus, is it possible to only provide SMILES string in input.py and run with Arkane and still obtain thermodynamics properties of the species? I only need heat of formation of the species and nothing else from Arkane.
